test/extended/builds/pipeline.go

```diff
@@ -100,6 +95,13 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 	})
 
 	g.Context("Pipeline with maven slave", func() {
+		g.AfterEach(func() {
+			if os.Getenv(jenkins.DisableJenkinsGCStats) == "" {
+				g.By("stopping jenkins gc tracking")
+				ticker.Stop()
+			}
+		})
+
```

Comment:
Interesting ... yes, that sounds like a good general practice to at least consider across our extended tests in general.

Thanks for the clarification.